packages/flutter_tools/test/src/fake_process_manager.dart

```diff
@@ -302,7 +302,9 @@ abstract class FakeProcessManager implements ProcessManager {
   }
 
   @override
-  bool canRun(dynamic executable, {String workingDirectory}) => true;
+  bool canRun(dynamic executable, {String workingDirectory}) => canRunSucceeds;
+
+  bool canRunSucceeds = true;
```

Comment:
PRobably not essential for this patch, but we might want to just have a `Set` of allowed things that canRun, which would help tests actually verify what's allowed to run.

Reply:
Added a `excludedExecutables` `Set` since all the tests I touched were explicitly forbidding executables.